alarm-frontend/src/Components/AddAlarm.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { minutesNumber, hourNumber } from "../fixNumbers";
+import useOption from "../Hooks/useOption";
+import axios from "axios";
+
+const BASE_URL = "http://127.0.0.1:5000";
```

Comment:
probably should think about how this will work when you need to deploy the app. Unless you're deploying both frontend and backend on the same server, this won't work.